lab_2_tokenize_by_bpe/main.py

```diff
@@ -36,6 +56,19 @@ def count_tokens_pairs(
     :param word_frequencies: dictionary in the form of <preprocessed word: number of occurrences>
     :return: dictionary in the form of <token pair: number of occurrences>
     """
+    if not isinstance(word_frequencies, dict):
+        return None
+    word_frequencies_list = list(word_frequencies)
+    print(word_frequencies_list)
+    dictionary = {}
+    for i in word_frequencies_list:
+        for j in range(len(i) - 1):
+            pair = (i[j], i[j + 1])
+            if pair not in dictionary:
+                dictionary[pair] = word_frequencies[i]
+            else:
```

Comment:
Consider rewriting it in such a way so that `else` is not needed

Reply:
fixed